hera_qm/firstcal_metrics.py

```diff
@@ -402,8 +371,8 @@ def __init__(self, calfits_files, use_gp=True):
 
         # Calculate avg delay solution and subtract to get delay_fluctuations
         self.delays = self.delays * 1e9
-        self.delay_avgs = np.median(self.delays, axis=1)
-        self.delay_fluctuations = (self.delays.T - self.delay_avgs).T
+        self.delay_avgs = np.median(self.delays, axis=1, keepdims=True)
+        self.delay_fluctuations = (self.delays - self.delay_avgs)
 
         # use gaussian process model to subtract underlying mean function
         if use_gp is True and self.sklearn_import is True:
```

Comment:
Do we want to expose the Gaussian process parameters/kernel choices to the user? Seems like things have been fine without doing that, but I didn't know if we should add these choices as kwargs.

Reply:
sounds like a good project for an interested grad student :+1: